Makefile

```diff
@@ -39,9 +39,13 @@ docker-run-cloud:
 
 # Runs Meshery on your local machine and points to locally-running  
 #  Meshery Cloud for user authentication.
+# SAAS_BASE_URL="http://mesherylocal.layer5.io:9876"
+	# SAAS_BASE_URL="https://staging-meshery.layer5.io"
+	# SAAS_BASE_URL="http://localhost:9876"
+
 run-local-cloud:
 	cd cmd; go clean; rm meshery; go mod tidy; go build -tags draft -a -o meshery; \
-	SAAS_BASE_URL="http://mesherylocal.layer5.io:9876" \
+	SAAS_BASE_URL="https://staging-meshery.layer5.io" \
```

Comment:
@kanishkarj, wouldn't it be better if we present a separate command for local meshery-cloud??
If we have a separate command for running using staging meshery, it would be great... Your opinions?

Reply:
Oh, definitely. I'm sorry, I actually ended up pushing the codebase as I tested it's working. By the way, it would be great if you and @Lee could try this PR out locally.
I'll we should have separate commands for staging and development. But what I was considering was just having different variables in make file corresponding to each SAAS_BASE_URL path. That might be more flexible.